tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/sparse-config/test-suite-a/testnamespace-testclass-testclassmethod-method.html

```diff
@@ -1,84 +1,47 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
   </head>
   <body>
     <section>
-      <h2>
-        testClassMethod
-      </h2>
+      <h2>testClassMethod</h2>
       <section>
-        <p>
-          Test class method
-        </p>
+        <p>Test class method</p>
       </section>
       <section>
-        <h3 id="testclassmethod-signature">
-          Signature
-        </h3>
-        <code>
-          testClassMethod(testParameter: string): Promise&lt;string&gt;;
-        </code>
+        <h3 id="testclassmethod-signature">Signature</h3><code>testClassMethod(testParameter: string): Promise&#x3C;string>;</code>
       </section>
       <section>
-        <h3 id="testclassmethod-parameters">
-          Parameters
-        </h3>
+        <h3 id="testclassmethod-parameters">Parameters</h3>
         <table>
           <thead>
             <tr>
-              <th>
-                Parameter
-              </th>
-              <th>
-                Type
-              </th>
-              <th>
-                Description
-              </th>
+              <th>Parameter</th>
+              <th>Type</th>
+              <th>Description</th>
             </tr>
           </thead>
           <tbody>
             <tr>
-              <td>
-                testParameter
-              </td>
-              <td>
-                <span>string</span>
-              </td>
-              <td>
-                A string
-              </td>
+              <td>testParameter</td>
+              <td><span>string</span></td>
+              <td>A string</td>
             </tr>
           </tbody>
         </table>
       </section>
       <section>
-        <h3 id="testclassmethod-returns">
-          Returns
-        </h3>
-        <p>
-          A Promise
-        </p>
-        <p>
-          <span><b>Return type:</b> </span><span>Promise&lt;string&gt;</span>
-        </p>
+        <h3 id="testclassmethod-returns">Returns</h3>
+        <p>A Promise</p>
+        <p><span><b>Return type: </b></span><span>Promise&#x3C;string></span></p>
```

Comment:
Again `&lt;` and `&gt;` mangling.

Reply:
Confirmed that this works correctly via ff.com build.
![image](https://github.com/user-attachments/assets/797a86f8-ca4d-46a0-b872-4889baff16e1)